sections/05-adc-data-publishing.qmd

```diff
@@ -64,8 +66,6 @@ Note that the dataset above lists a collection of files that are contained withi
 
 These data repositories all assign a unique identifier to ***every version*** of every data file, similarly to how it works with source code commits in GitHub. Those identifiers usually take one of two forms. A DOI (digital object identifier) identifier is often assigned to the metadata and becomes the publicly citable identifier for the package. Each of the other files gets a global identifier, often a UUID (universally unique identifier) that is globally unique. In the example above, the package can be cited with the DOI [`doi:10.5063/F1F18WN4`](@https://knb.ecoinformatics.org/view/doi:10.5063/F1F18WN4), and each of the individual files have their own identifiers as well.
 
-## Submitting Data from the Large Data Perspective
-
 ## Archiving Data: The Large Data Perspective
 
 There are two components to any data package archived with the Arctic Data Center: the metadata & the data themselves. Data can be images, plain text documents, tabular data, spatial data, scripts used to analyze the data, a readme file, and more. To the best of your ability, please make sure that the data uploaded are in an open format, rather than proprietary. This means uploading a `.pdf` rather than a `.docx` file, or `.csv` rather than `.xlsx`. That being said, we understand that this is not always possible or reasonable for users, as is the case sometimes if they've used ESRI products rather than QGIS.
```

Comment:
```suggestion
There are two components to any data package archived with the Arctic Data Center: the metadata & the data themselves. Data can be images, plain text documents, tabular data, spatial data, scripts used to analyze the data, a readme file, and more. To the best of your ability, please make sure that the data uploaded are in an open format, rather than proprietary format. We strongly recommend using open, self-documenting binary formats for large data archival. NetCDF, HDF, .mat (v7.3) and Parquet files are all examples of "self-documenting" files. In the case of a NetCDF file, users can input the attribute name, attribute description, missing value codes, units, and more into the file itself. When these data are well-documented within themselves, it can save the time when users submit their data to us, since the documentation for variable level information is already mostly complete. We'll discuss NetCDF and metadata more in Session 8. For geospatial data, we recommend using geotiff for raster files, and geopackage files for vector files.
```